configure/CONFIG_SITE

```diff
@@ -103,7 +103,7 @@
 # Which target architectures to cross-compile for.
 #  Definitions in configure/os/CONFIG_SITE.<host>.Common
 #  may override this setting.
-CROSS_COMPILER_TARGET_ARCHS=
+CROSS_COMPILER_TARGET_ARCHS=RTEMS-qoriq_e500
```

Comment:
Please remove this change from your PR. You can put that definition in a configure/CONFIG_SITE.local file in your working tree where it will have the same effect for you, but putting this in the PR breaks all our CI builds.

Reply:
Removed.